spec/core/collection_spec.rb

```diff
@@ -5,6 +5,58 @@
     ZendeskAPI::Collection.new(client, ZendeskAPI::TestResource)
   end
 
+  # TODO!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  # TODO: pager, xxx, url
```

Comment:
will clean up, and I also need to write at least one test for `Helpers#present?`